src/components/ec-modal/ec-modal.spec.js

```diff
@@ -72,6 +73,30 @@ describe('EcModal', () => {
     expect(wrapper.find('.ec-modal__content').element).toMatchSnapshot();
   });
 
+  it('should have the style attribute z-index when the z-index props is given', () => {
+    const wrapper = mountModal({
+      showModal: true,
+      zIndex: 210,
+    });
+
+    expect(wrapper.findByDataTest('ec-modal').attributes().style).toBe('z-index: 210;');
+    wrapper.setProps({ zIndex: 235 });
+    expect(wrapper.findByDataTest('ec-modal').attributes().style).toBe('z-index: 235;');
+    expect(wrapper.find('ec-modal').element).toMatchSnapshot();
```

Comment:
This yields no element and snapshot is undefined. You should always check what `toMatchSnapshot()` outputs to the snap file.